server.js

```diff
@@ -39,6 +41,19 @@ function warnIfAnalyticsNotSet () {
   }
 }
 
+function addCsrfMiddleware (app) {
+  app.use(csrf({
+    value: function (req) {
+      return req.body && req.body.csrfToken
```

Comment:
I suppose it's worth explicitly saying that the server will only be supporting CSRF validation through POST requests and will ignore headers/ query strings for now. That's in-line with the existing behaviour so looks like a good move.

Reply:
Yeah good call. Will add a comment in next PR to make it clear